partner/pms-common/src/main/java/io/mosip/pms/common/helper/SearchHelper.java

```diff
@@ -538,7 +538,8 @@ private SearchDto addPartnerFilter(SearchDto searchDto, String partnerIdColumn)
 	}
 	
 	public boolean isLoggedInUserFilterRequired() {
-		return !(UserDetailUtil.getLoggedInUserDetails().getAuthorities().stream()
-				.anyMatch(r -> r.getAuthority().equalsIgnoreCase("ROLE_" + partnerAdminRole)));
+		List<String> roles = Arrays.asList(partnerAdminRole.split(","));
```

Comment:
don't split with commas please use @value and store in a variable with List<String>dataType not required to split with commas

Reply:
changed as per the requirement